keanu-project/src/main/java/io/improbable/keanu/vertices/VertexState.java

```diff
@@ -0,0 +1,24 @@
+package io.improbable.keanu.vertices;
+
+import io.improbable.keanu.network.VariableState;
+import lombok.Value;
+
+import java.util.Optional;
+
+@Value
+public class VertexState<T> implements VariableState {
+    private T value;
+    private boolean isObserved;
+
+    public static final <T> VertexState<T> nullState() {
```

Comment:
what does `final` do when it's on a `static`?

Reply:
nothing... that was a hangover from when I originally had it as a field rather than a method.

removed.